templates/webapi/branding/openqa.suse.de/docbox.html.ep

```diff
@@ -2,9 +2,8 @@
 
 <p><a href="http://open.qa">» More information regarding openQA</a></p>
 <p><a href="https://confluence.suse.com/display/openqa/openQA">» Description of internal setup and administration</a></p>
-<p><a href="irc://irc.suse.de/testing">» IRC channel</a></p>
+<p><a href="https://chat.suse.de/channel/testing">» chat</a></p>
 <p>»
 	<a href="https://build.suse.de/project/staging_projects/SUSE:SLE-15:GA">SLE 15</a> and
-	<a href="https://build.suse.de/project/staging_projects/SUSE:SLE-12-SP3:Update:Products:CASP30">CaaSP</a> staging dashboards
```

Comment:
I suppose the previous point should then just be "SLE 15 staging dashboards" (and could be a single line).

Reply:
done